debian/bitlbee-common.config

```diff
@@ -1,21 +0,0 @@
-#!/bin/sh
-
-set -e
-
-. /usr/share/debconf/confmodule
-[ -f /etc/default/bitlbee ] && . /etc/default/bitlbee
-
-db_title BitlBee
-
-if [ -n "$BITLBEE_PORT" ]; then
```

Comment:
There, NEWS.Debian. That thing is decently loud, at least when upgrading.

I'm not sure about automatically migrating the port now. Editing the user config from scripts isn't going to be 100% reliable.